content/doc/developer/publishing/plugin-site.adoc

```diff
@@ -1,8 +1,109 @@
 ---
 title: Plugin Site
 layout: developersection
-wip: true
-references:
-- url: https://wiki.jenkins.io/display/JENKINS/Plugin+Site+Information+for+Plugin+Developers
-  title: Plugin site information on the wiki
 ---
+
+This page explains how https://plugins.jenkins.io/[the plugins site]
+works. The intended audience is plugin developers.
+
+This is not a manual on how to _use_ the site to e.g. find plugins, but
+rather explains how it gets its data and how plugin developers can
+change the data on it.
+
+== Data shown on the plugins site
+
+=== Which plugins are listed? How can I make my plugin appear?
+
+All plugins published to the current (weekly) update center are listed.
+
+=== What is the plugin installation trend based on?
+
+This uses the monthly data from http://stats.jenkins.io/ as gathered
+from Jenkins instances that have not opted out of reporting data.
+
+Learn more: link:../usage-statistics[usage statistics]
```

Comment:
Nit: I don't think this phrasing/lack of punctuation looks good when it's not a URL.